helm/zenko/charts/cloudserver-front/templates/deployment.yaml

```diff
@@ -28,6 +28,16 @@ spec:
               value: "{{- printf "%s-%s" .Release.Name "s3-metadata" | trunc 63 | trimSuffix "-" -}}"
             - name: REDIS_HOST
               value: "{{- printf "%s-%s" .Release.Name "redis" | trunc 63 | trimSuffix "-" -}}"
+{{- if index .Values "mongodb-replicaset" }}
+            - name: S3METADATA
+              value: mongodb
+            - name: MONGODB_HOST
+              value: {{- printf "%s-mongodb-replicaset.%s.svc.cluster.local" .Release.Name .Release.Namespace }}
+            - name: MONGODB_PORT
+              value: {{- template "mongodb-replicaset.port"  }}
```

Comment:
I'm surprised this works, because this seems to be inter-chart code, which is not supposed to be possible 😕

Reply:
I believe you are right and this isn't working how I intended necessarily. I will go back over this.